packages/teams-js/src/private/appEntity.ts

```diff
@@ -72,6 +73,10 @@ export namespace appEntity {
   ): void {
     ensureInitialized(FrameContexts.content);
 
+    if (!isSupported()) {
+      throw new Error(errorNotSupportedOnPlatform);
```

Comment:
Since the isSupported work started, we decided to throw SdkError instead of Error to be consistent.